app/src/main/res/layout-land/admin_pin_activity.xml

```diff
@@ -75,10 +75,12 @@
           style="@style/TextInputEditText"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
+          android:cursorVisible="true"
           android:imeOptions="actionNext"
           android:inputType="numberPassword"
           android:maxLength="5"
-          android:text="@{viewModel.savedPin}" />
+          android:text="@{viewModel.savedPin}"
+          android:textCursorDrawable="@null" />
```

Comment:
Why are we setting textCursorDrawable to null here?

Reply:
Setting textCursordrawable to null makes the cursor color to that of the text color(This is the same color that has to be set as the cursor color according to the themes). The reason I believe that the cursor was not appearing on focus is because it was covered by the cursor was covered by the border. One way to fix that would have been adding padding, but I feel this is a more elegant solution.

Reply:
Need to confirm with @rt4914 but I think a more elegant solution might be to add our own custom drawable for the cursor and then set it here.

Reply:
Sounds good @prayutsu Lets create a drawable.